### ISSUE TITLE
Clinvar pathogenic filter apparently does not work

### ISSUE THREAD

**Issue by holtgrewe** [bihealth/varfish-server#464]
**Describe the bug**
A user reported that filtering for "all clinvar pathogenic" variants does not work properly. We get both false positives and false negatives (pathogenic variants are not shown and non-pathogenic variants are shown).

**To Reproduce**
1. Filter case with "clinvar pathogenic" filter
2. See non-pathogenic variants and miss pathogenic variants.

**Expected behavior**
The filter results should reflect the clinvar flags that were used when annotating the case.

**Screenshots**
N/A

**Additional context**
This needs careful investigation. Earlier we had a discrepancy between the annotations within the Clinvar database and what was used for annotation. Now with the updated `20210728`(b) release the data should be consistent. We **can** expect inconsistencies with the upstream Clinvar database but that just cannot be helped.

**Comment by holtgrewe** [comment]
**Root Cause Analysis**
I'm looking into some filter results of the `in_clinvar` flag added by `varfish-annotator` and what is in the ClinVar table in VarFish Server.

The following are annotated as `in_clinvar` but have no corresponding entry in the `clinvar_clinvar` table.

```
 X          |    591926 | T                         | G                              | 
 X          |    591926 | T                         | G                              | 
 X          |  47306422 | T                         | C                              | 
 X          |  47327111 | T                         | C                              | 
 X          | 153640060 | CT                        | C                              | 
 X          | 153640060 | CT                        | C                              | 
```

Here is the query

```sql
SELECT v.chromosome, v.start, v.reference, v.alternative, c.variation_type
FROM variants_smallvariant AS v
LEFT JOIN clinvar_clinvar AS c
ON v.release=c.RELEASE
    AND v.chromosome=c.chromosome
    AND v.start=c.start
    AND v.reference=c.reference
    AND v.alternative=c.alternative
WHERE v.in_clinvar
    AND v.alternative != '*'
    AND v.case_id = 3892
    AND v.set_id = 5634;
```

**Comment by holtgrewe** [comment]
**Resolution Proposal**
- apparently no real bug fix needed in the queries
- display information in variant fold-out that the upstream clinvar database might have changed
- fix template such that the clinvar significance and rating is displayed properly

**Affected Components**
VarFish Server

**Affected Modules/Files**
Templates in `variants` module only.

**Required Architectural Changes**
None

**Required Database Changes**
None

**Backport Possible?**
Yes, should be packported to `anthenea`

**Resolution Sketch**
- [ ] display clinvar significance and rating again
- [ ] add informative note regarding possible difference to upstream clinvar
- [ ] fix the flag thing that we saw

The coincidental finding leads to filtering by flag to be broken in some way, should be fixed in #502, not here.
